.github/workflows/continuous_deployment.yml

```
@@ -28,6 +28,15 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata for Docker
+        id: meta
+        uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81
+        with:
+          images: ghcr.io/devoops23b/gosearch
+          tags: |
+            type=raw,value=latest
+            type=sha,format=short    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)